### PR TITLE
#9 Map Zone - Multiple CRS support

### PR DESCRIPTION
**Pull request by vlt1**
## Description
* Added new coordinate systems to localConfig
* Modified CRSSelector options

Issue: #9

### CODE REVIEW

File: localConfig.json [hunk 1]
```diff
@@ -326,7 +376,18 @@
...
+                    "label": "EPSG:3945"
+                  },
+                  "EPSG:3946": {
+                    "label": "EPSG:3946"
+                  },
+                  "EPSG:3948": {
+                    "label": "EPSG:3948"
+                  }
                 },
                 "filterAllowedCRS": [
```
Comment (mbarto) [inline localConfig.json]: Don't you need to add the new CRS codes here also?

Comment (vlt1) [reply localConfig.json]: @mbarto What do you mean?

Comment (mbarto) [reply localConfig.json]: Are the new CRS visible in the combo if they are not also listed in filterAllowedCRS?

Comment (vlt1) [reply localConfig.json]: additionalCRS is separate from filterAllowedCRS, it has not influence on it. For reference see [CoordinatesUtils](https://github.com/geosolutions-it/MapStore2/blob/43a979967a95895801d8f6cd9d30393532c082a8/web/client/utils/CoordinatesUtils.js#L429).

**Comment by vlt1** [comment]
Annotations seem to work as expected. With layer filters everything is fine until you try to edit the shape of a filter. Circle and rectangle shapes don't work correctly when you switch to one of the newly added projections. When you try to edit, for example, radius of a circle it disappears. With rectangles this problem persists even if one were to switch between the default projections(4326 and 3857).

**Pull request merged**